source/manual/create-a-local-transaction.html.md

```diff
@@ -99,7 +100,7 @@ end
 Now using the same pod as before and the file path from above, run the `kubectl cp` command to pull the file:
 
 ```bash
-$ gds aws govuk-integration-poweruser -- kubectl -n apps cp local-links-manager-fdc5dbbb7-vrf96:/app/tmp/1234.csv ~/Downloads/1234.csv
+$ kubectl -n apps cp local-links-manager-fdc5dbbb7-vrf96:/app/tmp/1234.csv ~/Downloads/1234.csv
```

Comment:
You can just write `/tmp` — `app/tmp` is a symlink to work around Ruby weirdness.

```suggestion
k cp local-links-manager-fdc5dbbb7-vrf96:/tmp/1234.csv ~/Downloads/1234.csv
```